templates/taxi/car_list.html

```diff
@@ -1,7 +1,13 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Car list</h1>
+  <h1>
+    Car list <br>
+  </h1>
+  <h3>
+    <a href="{% url 'taxi:car-create' %}">Create 🚕</a>
```

Comment:
I think it's better to add `class="btn btn-primary"` to make the element look like a button